mlflow/projects/_project_spec.py

```diff
@@ -27,6 +27,7 @@ def load_project(directory):
         project_name = None
     conda_path = yaml_obj.get("conda_env")
     docker_env = yaml_obj.get("docker_env")
+    kubernetes_env = yaml_obj.get("kubernetes_env")
```

Comment:
Do we still use `kubernetes_env`, if not let's remove all usages of it.

Reply:
Sorry, we are not using it anymore.

Reply:
done!